share/picongpu/tests/compileAtomicPhysics/include/picongpu/param/particle.param

```diff
@@ -46,19 +46,6 @@ namespace picongpu::particles
              */
             static constexpr uint32_t numParticlesPerCell = 10u;
         };
-
-        struct OnePositionParameter
-        {
-            /** Count of particles per cell at initial state
-             *  unit: none
-             */
-            static constexpr uint32_t numParticlesPerCell = 1u;
-            // sit directly in lower corner of the cell
-            // each x, y, z in-cell position component in range [0.0, 1.0)
-            static constexpr auto inCellOffset = float3_X(0.0, 0.0, 0.0);
-        };
-        using OnePosition = OnePositionImpl<OnePositionParameter>;
-        using Random10ppc = RandomImpl<RandomParameter10ppc>;
```

Comment:
still required

Reply:
@BrianMarre is right about that. The `Random10ppc` is used [here](https://github.com/ComputationalRadiationPhysics/picongpu/blob/2572a435a2b0a1e2121ee12cd885a9817ea0c571/share/picongpu/tests/compileAtomicPhysics/include/picongpu/param/speciesInitialization.param#L44). It is surprising for me that the CI still passes as this should cause a compile error. Do we execute this test?

Reply:
re-added it